model/publishing/delivery/RemoteDeliveryPublisher.php

```diff
@@ -110,18 +134,26 @@ private function prepareRequestData(): array
             ];
 
             $deliveryClass = current($this->delivery->getTypes());
-            if ($deliveryClass->getUri() != DeliveryAssemblyService::CLASS_URI) {
+
+            if ($deliveryClass->getUri() !== DeliveryAssemblyService::CLASS_URI) {
+                $labels = $this->getParentLabels($deliveryClass);
+
+                $labels[] = $deliveryClass->getLabel();
+
                 $requestData[] = [
-                    'name' => RestTest::REST_DELIVERY_CLASS_LABEL,
-                    'contents' => $deliveryClass->getLabel()
+                    'name' => RestTest::REST_DELIVERY_CLASS_LABELS,
+                    'contents' => json_encode($labels),
```

Comment:
What if instead of introducing a breaking change in the API we've added another property here?
We can deprecate the original one and eventually have it removed then.

Reply:
Done [here](https://github.com/oat-sa/extension-tao-delivery-rdf/pull/360/files#diff-0cd00b39eb8dd0f162481b8468f53043R52).

Reply:
Almost, the constant is there, but the code doesn't read the provided parameter.
Let's fall back to it in case `REST_DELIVERY_CLASS_LABELS` are not provided.

Reply:
https://github.com/oat-sa/extension-tao-delivery-rdf/pull/360/commits/7aa816e28bcd35b9a3051b770335e72ae6f2e39b